packages/home-spa/.gitignore

```diff
@@ -41,6 +41,12 @@ dist/
 *.key
 *.pem
 
+# ignore environment files
+.env
+.env.production
+.env.local
+.env.development 
```

Comment:
instead of `.env.production, .env.local, .env.development `. Can you please use conventions like `production.env, local.env, development.env` If you follow this you will need to add only .env in the `.gitignore` files

Reply:
This is perfect. I'll do it.